yax/artifacts/informative_alignment_data.py

```diff
@@ -0,0 +1,21 @@
+from yax.state.type.artifact import Artifact
+from yax.shared.utilities import taxidtool
+
+
+class InformativeAlignmentData(Artifact):
+    def __init__(self, completed):
+        self.tax_id_data = None
+        self.inclusion_tree = None
+        self.gis_to_taxids = None
+        self.sequences_input_fp = None
+        self.truncation_level = None
```

Comment:
Unclear what any of these are. A docstring an be added to the `class` itself with a `Properties` section.